imap_processing/codice/codice_l1a.py

```diff
@@ -360,14 +366,25 @@ def create_event_dataset(
     dataset : xarray.Dataset
         Xarray dataset containing the event data.
     """
+    if apid == CODICEAPID.COD_LO_PHA:
+        dataset_name = "imap_codice_l1a_lo_pha"
+    elif apid == CODICEAPID.COD_HI_PHA:
+        dataset_name = "imap_codice_l1a_hi_pha"
+
+    # Determine the start time of the packet
+    met = packet.acq_start_seconds.data[0]
+
+    # Extract the data
+    # event_data = packet.event_data.data (Currently turned off, see TODO)
+
     cdf_attrs = ImapCdfAttributes()
     cdf_attrs.add_instrument_global_attrs("codice")
     cdf_attrs.add_instrument_variable_attrs("codice", "l1a")
     cdf_attrs.add_global_attribute("Data_version", data_version)
 
     # Define coordinates
     epoch = xr.DataArray(
-        met_to_j2000ns(met),  # TODO: Fix after SIT-3 (see note below)
+        met_to_j2000ns([met]),
```

Comment:
This is already calculated in the dataset routine. So `epoch` in the incoming dataset should be what you want here and just need to add on the attributes like you're doing.

```suggestion
        packet["epoch"],
```